jmx-metrics/src/main/resources/target-systems/hadoop.groovy

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+def beanHadoopNameNodeFS = otel.mbean("Hadoop:service=NameNode,name=FSNamesystem")
+otel.instrument(beanHadoopNameNodeFS, "hadoop.hdfs.disk.usage", "The amount of disk used by data nodes.", "by",
+  ["node_name" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "CapacityUsed", otel.&longUpDownCounterCallback)
+otel.instrument(beanHadoopNameNodeFS, "hadoop.hdfs.disk.limit", "The total disk allotted to data nodes.", "by",
+  ["node_name" : { mbean -> mbean.getProperty("tag.Hostname") }],
+  "CapacityTotal", otel.&longUpDownCounterCallback)
+otel.instrument(beanHadoopNameNodeFS, "hadoop.hdfs.block.count", "The total number of blocks.", "{blocks}",
```

Comment:
I don't think `{}` are common in otel units so would prefer we not use them.

Reply:
Oh nice thanks for the head's up. I had missed this update and assumed it was from porting similar scripts.